src/objects.js

```diff
@@ -1,41 +1,74 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name: name,
+          age: age}
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]){
```

Comment:
Nice, there is also a method called Object.hasOwnProperty(). It returns a boolean, so you could refactor this to be one line. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty